exporter/opentelemetry-exporter-jaeger-thrift/proto/api_v2/collector.proto

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2019 The Jaeger Authors.
```

Comment:
These proto definition shouldn't be needed for the thrift package right?

Reply:
Thanks, removed